### DISCUSSION TITLE
Apollo Save Tool v2.0.2

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps3#164] [General]
### Added

* Delete PS3 saves from USB
* PS2 Virtual Memory Card images (VMC)
  - Support additional import formats: `.XPS`, `.CBS`, `.MAX`, `.SPS`
* Online DB: show PS1 icons
* User Tools:
  - Convert PS1/PS2 saves to `.PSV` format
  - Convert `.PSV` saves to `.MCS`/`.PSU` formats
  - Re-sign `.PSV` saves

### Fixed

* "Copy to HDD" option is only available when browsing USB saves

### Misc

* Updated [Apollo Patch Engine](https://github.com/bucanero/apollo-lib) to v1.1.2
  - Fixed SW Code search bug when bytes are not found
* Option to launch an external app
  - (L1 + START) executes `/dev_hdd0/game/NP0APOLLO/USRDIR/EXTRA.SELF`

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps3/releases/tag/v2.0.2'>Apollo Save Tool v2.0.2</a>.</em>

**Comment by Gabbie-S** [comment]
What does "Option to launch an external app
(L1 + START)" do?

**Reply by bucanero** [reply]
as stated above, that combo will execute an external file, `/dev_hdd0/game/NP0APOLLO/USRDIR/EXTRA.SELF` . The file is anything that the user wants to execute. It could be another app, a file manager, etc.

It's only meant for advanced users who want to jump from Apollo to another homebrew quickly.
